app/controllers/my-services/get-index-controller.js

```diff
@@ -0,0 +1,54 @@
+const q = require('q')
```

Comment:
JS noob question: do we really need to use `q` here, when (I seem to remember) we decided to switch to native promises whenever possible?